homeassistant/components/zha/core/helpers.py

```diff
@@ -421,3 +419,30 @@ def qr_to_install_code(qr_code: str) -> tuple[zigpy.types.EUI64, bytes]:
         return ieee, install_code
 
     raise vol.Invalid(f"couldn't convert qr code: {qr_code}")
+
+
+@dataclasses.dataclass(kw_only=True)
```

Comment:
```suggestion
@dataclasses.dataclass(kw_only=True, slots=True)
```

https://docs.python.org/3/library/dataclasses.html#dataclasses.dataclass

We can probably use slots here since py3.11+ is required